src/Components/SpecialContent/SpecialContent.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { withRouter } from "react-router-dom";
+// import { itemList } from "./itemListData";
+import "./SpecialContent.scss";
+
+class SpecialContent extends React.Component {
+  state = {
+    dDay: 0,
+  };
+
+  componentDidMount() {
+    this.timer = setInterval(() => this.tick(), 1000);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.timer);
+  }
+
+  tick() {
+    this.setState({
+      dDay: new Date("2020/07/31 00:00:00").getTime() - new Date().getTime(),
+    });
+  }
+
+  digitCheck = (time) => {
+    return time < 10 ? `0${time}` : time;
+  };
+
+  render() {
+    const { dDay } = this.state;
+    const { digitCheck } = this;
+    const { index, content } = this.props;
+    console.log(index);
+    console.log(dDay);
+    return (
```

Comment:
```suggestion

    return (
```
콘솔 로그 제거하고 한줄 띄워주세요.

Reply:
수정완료